billing-ui/src/pages/Report/aggregateEntries.js

```diff
@@ -44,12 +44,14 @@ const addNumbers = (acc, value, key) => {
 const aggregateEntries = (entries, groupByIteratee) =>
     Object.entries(groupBy(entries, groupByIteratee))
         .map(([key, items]) => items.reduce((prevItems, entry) => ({
-            ...prevItems,
             ...entry,
             ...FIELDS.reduce((prevFields, fieldName) => ({
                 ...prevFields,
                 ...addNumbers(prevFields, entry, fieldName),
-            }), { key }),
+            }), {
+                key,
+                ...prevItems,
```

Comment:
I was merging the objects prematurely, and they were getting overwritten instead of summed up.